cmd/create.go

```diff
@@ -35,6 +34,10 @@ var createCmd = &cobra.Command{
 	Use:   "create",
 	Short: "Create the resource on the Kubernetes cluster",
 	Run: func(cmd *cobra.Command, args []string) {
+		if err := ifFilesPassed(CreateFiles); err != nil {
+			fmt.Println(err)
```

Comment:
OK, we can leave it without help for now.
Later we can look into kubectl how they do it.